docker-compose.yml

```diff
@@ -11,7 +11,7 @@ services:
       - .:/app
     working_dir: /app
     command: >
-      /bin/sh -c "npm install && npm run start:dev"
+      /bin/sh -c "corepack enable && pnpm install && pnpm start:dev"
```

Comment:
Shouldn't we also run `corepack prepare pnpm@latest-10` here?

Reply:
You are right, fixed here!